latex/method.tex

```diff
@@ -35,18 +37,21 @@ \subsection{Computing Delivered Fluence}
 \end{equation}
 
 Each of these integrals can then be computed using standard quadrature techniques.
-Here we choose to use Simpson quardature.
+Here we choose to use Simpson quadrature.
 
 \subsection{template:}
 
-We consider a single row version of the problem. Extension to the full fluence map should be possible, but it makes sense
-to first develop the method and see how it works for a single row. Let $f(x)$ be the desired fluence map to deliver, where
-$x$ is the position across the fluence row, $x \in [-1,1]$. We choose this as the domain because that is the
-domain that the Chebyshev polynomials are written for. Across that domain the Chebyshev polynomials range from $-1$ to $1$.
+We consider a single row version of the problem. 
+Extension to the full fluence map should be possible, but it makes sense to first develop the method and see how it works for a single row. 
+Let $f(x)$ be the desired fluence map to deliver, where $x$ is the position across the fluence row, $x \in [-1,1]$. 
+We choose this as the domain because that is the domain that the Chebyshev polynomials are written for. 
+Across that domain the Chebyshev polynomials range from $-1$ to $1$.
 We assume we have $T=1$ time unit to reproduce the fluence map $f$ by sliding leaves across the field from left to right.
-Let $v_L(t)$ be the velocity of the left leaf at time $t$ and $v_R(t)$ the right leaf velocity. We assume we fix the starting positions
-of both leaves. We may end up assuming we fix the ending positions too. This is not ideal, in fact we would like to optimize over
-starting and ending leaf positions as well, but to make things more concrete, for now starting positions are assumed fixed and given.
+Let $v_L(t)$ be the velocity of the left leaf at time $t$ and $v_R(t)$ the right leaf velocity. 
+We assume we fix the starting positions of both leaves. 
+We may end up assuming we fix the ending positions too. 
+This is not ideal, in fact we would like to optimize over starting and ending leaf positions as well, but to make things more concrete, for now starting positions are assumed fixed and given.
+\KvAcomment{I'm curious about the start/end position search. I'm using 2 methods: one uses restrictive sampling, the other partial enumeration. Both assume unidirectional trajectories though.}
```

Comment:
This section was written by David, and doesn't totally match the Matlab code. Right now I allow both the starting and ending positions to be arbitrary, although we modify either to be fixed without too much work.